internal/provisioner/main.go

```diff
@@ -91,6 +96,19 @@ func main() {
 		os.Exit(0)
 	}
 
+	args := flag.Args()
+	if len(args) != 1 {
```

Comment:
TODO: this probably should be overhauled to use cobra, so we can enforce these types of semantics before runtime

Reply:
That is fair, Joe mentioned that and I'll work on adding it in.

Reply:
I think it's a good candidate for a follow-up